### PR TITLE
[AM][REFACTOR-27][FEAT-36]: Fix lint errors, rewrite styles

### PR DESCRIPTION
**Pull request by tiescl**
## Warn: no new functionality was added, only error fixing and changes to styling.

### CODE REVIEW

**Review by bek-shoyatbek — APPROVED** [review]

File: src/modules/auth/styles/input.sx.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+export const inputStyles = (focused: boolean, hasContent: boolean, noBorder: boolean = true) => ({
+  container: {
+    position: 'relative',
+    height: '57px',
+    borderBottom: '1px solid rgb(176, 176, 176)',
+    ...(focused && {
```
Comment (MdRasulov) [inline src/modules/auth/styles/input.sx.ts:6]: why not just use `&:focus` here and below that?

File: src/modules/auth/components/login/components/email-input.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const { ref, ...rest } = register('email');
+
+  return (
+    <Box sx={styles(focusedField === 'email', false, focusedField === 'password').container}>
+      <Box sx={styles(focusedField === 'email', !isInputFieldEmpty(emailRef)).label}>Email</Box>
+
+      <InputBase
+        {...rest}
+        autoCorrect="off"
+        ref={(e: HTMLDivElement | null) => {
```
Comment (MdRasulov) [inline src/modules/auth/components/login/components/email-input.tsx:19]: I see a lot of repetition of this, lets create a separate function for that and maybe reuse it in the inputs too

**Review by MdRasulov — CHANGES_REQUESTED** [review]

File: src/modules/auth/components/login/login-form/login-form.sx.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+
+    '&::before': {
+      content: "''",
+      display: 'block',
+      width: '100%',
+      height: '1px',
+      backgroundColor: '#dddddd',
```
Comment (MdRasulov) [inline src/modules/auth/components/login/login-form/login-form.sx.ts:24]: make sure that mostly used colors are stored as constants

File: src/modules/auth/components/login/components/email-input.tsx (hunk 2 above)
Comment (tiescl) [reply src/modules/auth/components/login/components/email-input.tsx]: fixed

File: src/modules/auth/components/login/login-form/login-form.sx.ts (hunk 3 above)
Comment (tiescl) [reply src/modules/auth/components/login/login-form/login-form.sx.ts:26]: fixed

File: src/modules/auth/styles/auth-page.sx.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
+const authPageStyles = {
```
Comment (bek-shoyatbek) [inline src/modules/auth/styles/auth-page.sx.ts]: Could you rename your style files to .styles.ts

**Review by bek-shoyatbek — CHANGES_REQUESTED** [review]

**Comment by farrooh** [comment]
Don't forget to resolve conversations after completing them.

File: src/modules/auth/styles/auth-page.sx.ts (hunk 4 above)
Comment (tiescl) [reply src/modules/auth/styles/auth-page.sx.ts]: ok

**Review by bek-shoyatbek — APPROVED** [review]

File: src/modules/auth/constants/styles.constant.ts
Comment (farrooh) [inline src/modules/auth/constants/styles.constant.ts:1]: Are these constants specific only to the auth module? If they're the same throughout the entire project, please move this into a shared folder. For example, primaryBorderColor is the same on every page, I suppose?

Comment (tiescl) [reply src/modules/auth/constants/styles.constant.ts:1]: Honestly, I don't know, our frontend is not very structured and everyone is just doing their part with their own approach. I think its best to keep it inside auth module

**Review by MdRasulov — APPROVED** [review]

**Pull request merged**